VirtoCommerce.Platform.Web/Converters/Diagnostics/InstalledModuleInfoConverter.cs

```diff
@@ -0,0 +1,26 @@
+using System.Linq;
+using Omu.ValueInjecter;
+using VirtoCommerce.Platform.Web.Model.Diagnostics;
+using VirtoCommerce.Platform.Web.Model.Modularity;
+using coreModel = VirtoCommerce.Platform.Core.Modularity;
+
+namespace VirtoCommerce.Platform.Web.Converters.Diagnostics
+{
+    public static class InstalledModuleInfoConverter
```

Comment:
If we reused this exits [ModuleDescriptor](https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Model/Modularity/ModuleDescriptor.cs) we wouldn't need to define the new converter and would use this one [ModuleDescriptorConverter](https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Converters/Modularity/ModuleDescriptorConverter.cs)

Reply:
I agree.